pakyow-core/lib/core/middleware/req_path_normalizer.rb

```diff
@@ -0,0 +1,27 @@
+require 'core/call_context'
+
+module Pakyow
```

Comment:
@bryanp, I thought I'd go old school and manual test this repo so I hooked my project with this repo, like:
```
gem 'pakyow', '~> 0.10', :git => 'https://github.com/karimmtarek/pakyow', :branch => 'handel-request-path-with-double-slash'
```
And when I tried `pakyow server` that's what I got:
```
Puma caught this error: undefined method `new' for #<Pakyow::Middleware::ReqPathNormalizer:0x007f9ffa05b9e0> (NoMethodError)
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:86:in `block in use'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:147:in `block in to_app'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:147:in `each'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:147:in `inject'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:147:in `to_app'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/builder.rb:153:in `call'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/puma-2.15.3/lib/puma/server.rb:541:in `handle_request'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/puma-2.15.3/lib/puma/server.rb:388:in `process_client'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/puma-2.15.3/lib/puma/server.rb:270:in `block in run'
/Users/karim/.rvm/gems/ruby-2.3.0/gems/puma-2.15.3/lib/puma/thread_pool.rb:106:in `block in spawn_thread'
```
What do you think?